rdkit/Chem/UnitTestMol3D.py

```diff
@@ -443,14 +431,32 @@ def testEnumerateEitherDoubleStereo(self):
     rdbase = os.environ["RDBASE"]
     filename = os.path.join(rdbase, 'Code/GraphMol/FileParsers/test_data/simple_either.mol')
     mol = Chem.MolFromMolFile(filename)
-    smiles = [Chem.MolToSmiles(m) for m in AllChem.EnumerateStereoisomers(mol)]
+    smiles = [Chem.MolToSmiles(m) for m in EnumerateStereoisomers(mol)]
     self.assertEqual(set(smiles), {"C/C=C/C", "C/C=C\\C"})
 
   def testTryEmbeddingManyChirals(self):
     smiles = "C1" + "C(Cl)(Br)" * 40 + "C1"
     mol = Chem.MolFromSmiles(smiles)
-    opts = AllChem.StereoEnumerationOptions(tryEmbedding=True, maxIsomers=2)
-    self.assertEqual(len(list(AllChem.EnumerateStereoisomers(mol, options=opts))), 2)
+    opts = StereoEnumerationOptions(tryEmbedding=True, maxIsomers=2)
+    self.assertEqual(len(list(EnumerateStereoisomers(mol, options=opts))), 2)
+
+  def testShorcutImport(self):
+    """ Borrow test in testTryEmbeddingManyChirals 
+        See here if needed: https://stackoverflow.com/questions/4319825/python-unittest-opposite-of-assertraises
+
+    """
+    smiles = "C1" + "C(Cl)(Br)" * 40 + "C1"
+    mol = Chem.MolFromSmiles(smiles)
+    try:
+      opts = AllChem.StereoEnumerationOptions(tryEmbedding=True, maxIsomers=2)
+    except AttributeError as e:
+      self.fail(e)
+    
+    try:
+      tempV = AllChem.EnumerateStereoisomers(mol, options=opts)
+      self.assertEqual(len(list(tempV)), 2)
+    except AttributeError as e:
+      self.fail(e)
```

Comment:
Where is this test coming from?

Reply:
As mentioned earlier, this came from the `import`. This test is important as we want these two functions can be accessed shortcut on `AllChem.py` but not only `Chem.EnumerateStereoisomers.py`. This is to prevent people try to delete the shortcut import on `AllChem.py`. If it is redundant, I can delete it.

Reply:
If you dig out `AllChem.py`, you would see that a numerous of function has been imported but not use in the module.

Reply:
I get it. I wonder if there is a simpler test. One could for example do the following:
```
from rdkit import Chem
from rdkit.Chem import AllChem
chem = set(Chem.__dict__)
allchem = set(AllChem.__dict__)

expectedMissing = {'AllChem'}
chem = chem - expectedMissing
missing = [k for k in chem if k not in allchem and not k.startswith('_')]
... filter list of functions contained in chem but not available in allchem ...
missing = 
assert not missing
```
Currently this returns:
```
>>> missing
['rdMolAlign', 'rdDepictor', 'rdMolTransforms', 'rdReducedGraphs', 'rdPartialCharges', 'rdChemReactions', 'rdSLNParse', 'rdShapeHelpers', 'rdMolEnumerator', 'rdForceFieldHelpers', 'ChemicalFeatures', 'rdMolChemicalFeatures', 'rdDistGeom', 'rdMolDescriptors', 'rdChemicalFeatures', 'rdqueries']
```
@greglandrum looks like not all of the functionality is exported by `AllChem`. Would it make sense to add a test like this to make sure that new functionality is also added to `AllChem`?

Reply:
> here is a simpler test that could be added to `UnitTestAllChem`. One could for example do the following:

I don't recommend this test, as a slight change in `AllChem` but not in `Chem` would break the test and everyone has to add it in the module. Meanwhile, although it has stated that `AllChem` is the extension of `Chem` but aimed at the more complex operation. I doubt that when we add functions into `AllChem`, this could be break

Reply:
If you want to make sure that functionality is exposed through `AllChem` and `Chem` then having a test in `UnitTestAllChem` that does this is a reasonable approach. Adding functionality to `AllChem` would not break the suggested test, but adding functionality to `Chem` would. At this point, one would need to address this situation either by including the export into `AllChem` or by adding the new functionality to the `expectedMissing` list. In this case, breaking the test is what you want to have.

Reply:
I think the better case is to delete the test I have made. This would be better.

Reply:
I agree. I'll point Greg to this discussion to see if he thinks adding the AllChem test (and potentially adding the missing imports) would be useful.

Reply:
The original intent of AllChem is that it should bring in all the extension modules. And, you're right that it no longer does so.
I think fixing this and adding the test to automatically catch future problems is a great idea.

Reply:
> The original intent of AllChem is that it should bring in all the extension modules. And, you're right that it no longer does so. I think fixing this and adding the test to automatically catch future problems is a great idea.

I agree, but I don't have enough time to check every extension is correctly imported into `AllChem.py`. Eventhough it could, I have seen tons of `AllChem.MolFromSmiles()` and similar function which should not be called but should on `Chem.py`.

Reply:
> I agree, but I don't have enough time to check every extension is correctly imported into `AllChem.py`.

@gedeck showed an example above of how to do this.

Reply:
> > I agree, but I don't have enough time to check every extension is correctly imported into `AllChem.py`.
> 
> @gedeck showed an example above of how to do this.

Thanks @gedeck, but I believe this beyond the scope of this PR, I will make a second one to tackle with this. I had also revert the type hint in `__init__.py` back to original. I also switch the `LegacyImplementation` of `FindMolChiralCenters to different function. This can help another developers understand and did not make any unintentional violation